libraries/unit_converter.rb

```diff
@@ -0,0 +1,44 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# File Name:: unit_converter.rb
+#
+# Module contains different functions used to manipulate the units of file sizes.
+#
+module CernerSplunk
+  SIZE_SCALE = %w(KB MB GB TB).freeze
+  # Methods converts file sizes in KB, MB, GB and TB into Bytes.
+  def self.convert_to_bytes(string) # rubocop:disable CyclomaticComplexity
+    string = string.sub(/\s+/, '')
+    # return the floor value of string converted into float if specified without units - bytes
+    return string.to_f.floor if string =~ /^\d*\.?\d+$/
+    units = %w(B K KB KiB M MB MiB G GB GiB T TB TiB)
+    fail "Unparsable size input: #{string}" unless units.include? string.scan(/^\d*\.?\d+|\w+/)[1]
```

Comment:
Even though examples are shown in the databag docs on how to specify the quota, I decided to make it case insensitive just to be safe.
2f785dd7d6d0d2fbecc029717595ae8fc9f17ed4